css-positioning/relative.css

```diff
@@ -0,0 +1,47 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+  display: flex;
+}
+
+.pokemon > img {
+  width: 150px;
+  flex-basis: 20%;
+
+  /* your code here */
+}
+
+#bulbasaur {
+  /* your code here */
+  position: relative;
+  right: 200px;
+}
+
+#charmander {
+  /* your code here */
+  position: relative;
+  top: 50px;
+  right: 200px;
+}
+
+#squirtle {
+  /* your code here */
+  position: relative;
+  top: 100px;
+  right: 200px;
+}
+
+#pikachu {
+  /* your code here */
+  position: relative;
+  top: 200px;
+  right: 200px;
+}
+
+#jigglypuff {
+  /* your code here */
+  position: relative;
+  top: 300px;
+  right: 200px;
+}
```

Comment:
What ruleset do we have that we could use to apply `position: relative` to all of the images rather than adding it to each ruleset? You also shouldn't need the `right: 200px` here, just need to push it from the top.